Client/src/Components/Graph/GraphControls/AxesControl.tsx

```diff
@@ -26,10 +26,59 @@ const buildVariableList = (datasets: IDatasetModel[]): IVariable[] => {
       }
     })
   })
-
   return variables
 }
 
+const getVariableDimension = (datasets: IDatasetModel[], variableName: string): number => {
```

Comment:
Nice! This is valid refactoring... I dont remember exactly what its called. I think extract method maybe? Maybe check and note this one. Will go into our report